src/org.xtuml.bp.core/src/org/xtuml/bp/core/common/PersistenceManager.java

```diff
@@ -810,20 +811,21 @@ static public List findAllComponents(ModelRoot modelRoot, Class elementClass) {
 
           IPath path = null;
 
-          // for the compare and clipboard roots return an empty list
```

Comment:
With the batchRelate calls now passing a null model root (to use all roots in the workspace), the section for the compare case is moved lower within a not null  test.  Starting at line 821.